core/bin/zksync_server/src/main.rs

```diff
@@ -197,13 +197,6 @@ async fn main() -> anyhow::Result<()> {
         _ = sigint_receiver => {
             tracing::info!("Stop signal received, shutting down");
         },
-        error = cb_receiver => {
-            if let Ok(error_msg) = error {
-                let err = format!("Circuit breaker received, shutting down. Reason: {}", error_msg);
-                tracing::warn!("{err}");
-                vlog::capture_message(&err, vlog::AlertLevel::Warning);
-            }
-        },
```

Comment:
to add circuit breaker for the framework we have to change old logic.
lets avoid using channel and instead will send the error from the circuit breaker `check` method, in case the check fails. in fact the reaction of the system on the failed check won't change  since `wait_for_tasks()` runs circuit breaker task anyway.